packages/react-data-grid/src/masks/InteractionMasks.js

```diff
@@ -95,6 +92,39 @@ class InteractionMasks extends React.Component {
     firstEditorKeyPress: null
   };
 
+  saveEditorPosition = () => {
+    if (this.selectionMask) {
+      const { left, top } = this.selectionMask.getBoundingClientRect();
+      const { scrollLeft, scrollTop } = document.documentElement;
+      this.editorPosition = { left: left + scrollLeft, top: top + scrollTop };
```

Comment:
I noticed you didn't use this `window.pageYOffset` in the `transform` function that I added in the merged PR. Is it because of the `this.selectionMask` ref? Because the original PR added a window scroll event listener, so that when the grid is at far bottom with window scroll bar and the user drags the window level scroll bar, the mask position will be refreshed. For this `InteractionMasks`, will the position be refreshed when the window level scroll bar is dragged?

Reply:
`window.pageYOffset` is not needed anymore as the editor is now absolutely positioned instead of fixed so it will just work. We however need to make sure scrollleft works for frozen columns and `setScrollLeft` handles that logic

Reply:
👍